File: python/1013.py
# Read three integers
a, b, c = map(int, input().split())

# Function to find greatest of two numbers
def maior(x, y):
    return (x + y + abs(x - y)) // 2

# Apply formula
maior_ab = maior(a, b)
maior_abc = maior(maior_ab, c)

# Output result
print(f"{maior_abc} eh o maior")

File: javascript/beginner/1013.js
// Read input (Node.js environment)
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').trim().split(/\s+/).map(Number);

// Get the three values
const a = input[0];
const b = input[1];
const c = input[2];

// Function to find the greatest using the formula
function maior(x, y) {
    return (x + y + Math.abs(x - y)) / 2;
}

// Apply the formula
let m = maior(a, b);
m = maior(m, c);

// Output result
console.log(`${m} eh o maior`);